shared/clientstats/interfaces.go

```diff
@@ -0,0 +1,11 @@
+package clientstats
+
+import "io"
+
+type Scraper interface {
```

Comment:
Comments on these about what they are and how they're used would be helpful

Reply:
updated w/ comments